controllers/v1beta1/build_deletionhandlers.go

```diff
@@ -213,23 +213,29 @@ Build cancelled
 		}
 		// send any messages to lagoon message queues
 		// update the deployment with the status
-		r.cancelledBuildStatusLogsToLagoonLogs(ctx, opLog, &lagoonBuild, &lagoonEnv)
-		r.updateCancelledDeploymentAndEnvironmentTask(ctx, opLog, &lagoonBuild, &lagoonEnv)
-		r.cancelledBuildLogsToLagoonLogs(ctx, opLog, &lagoonBuild, allContainerLogs)
+		r.buildStatusLogsToLagoonLogs(ctx, opLog, &lagoonBuild, &lagoonEnv, true)
+		r.updateDeploymentAndEnvironmentTask(ctx, opLog, &lagoonBuild, &lagoonEnv, true)
+		r.buildLogsToLagoonLogs(ctx, opLog, &lagoonBuild, allContainerLogs, true)
 	}
 	return nil
 }
 
-// cancelledBuildLogsToLagoonLogs sends the build logs to the lagoon-logs message queue
+// buildLogsToLagoonLogs sends the build logs to the lagoon-logs message queue
 // it contains the actual pod log output that is sent to elasticsearch, it is what eventually is displayed in the UI
-func (r *LagoonBuildReconciler) cancelledBuildLogsToLagoonLogs(ctx context.Context,
+func (r *LagoonBuildReconciler) buildLogsToLagoonLogs(ctx context.Context,
 	opLog logr.Logger,
 	lagoonBuild *lagoonv1beta1.LagoonBuild,
 	logs []byte,
+	cancelled bool,
 ) {
 	if r.EnableMQ {
-		condition := "cancelled"
-		buildStep := "cancelled"
+		condition := "pending" //@TODO: remove this and uncomment the queued condition when lagoon supports queued status
```

Comment:
Just as an aside, this is actually a really good example of where I've seen feature flags been used to good effect.
This change is wrapped in a feature flag, and then simply enabled when the target feature on the external service is rolled out. This could be, for instance, linked to the core lagoon version or something.
Not saying we need to change anything, it just struck me as a good example of that pattern.

Reply:
I like the idea, will keep in mind for future changes. I didn't do it this time since its such a small thing to fix in a later version.